api/opencl_runtime_layer.asciidoc

```diff
@@ -8565,7 +8562,7 @@ include::{generated}/api/version-notes/clGetEventInfo.asciidoc[]
 
 [[event-info-table]]
 .List of supported param_names by <<clGetEventInfo>>
-[width="100%",cols="<30%,<33%,<37%",options="header"]
+[width="100%",cols="<33%,<17%,<50%",options="header"]
 |====
 | Event Info | Return Type | Info. returned in _param_value_
```

Comment:
This is very minor, but I noticed it yesterday in other `clGetFooInfo` entry-points. Should the period after "Info" exist in the third column heading? Think it reads better without it. Feel free not to make this change now even if so to make any spec release time constraints

Reply:
Thanks, good suggestion.  I've changed this to use "Description" as the column header consistently, which was already a convention used by other table cells.  If we want to use a more descriptive column header in the future this is easy enough to change, but for now I think the consistency is a nice improvement.  Thanks again!